stylelint-config.json

```diff
@@ -1,5 +1,59 @@
 {
   "rules": {
+    "color-hex-case": "lower",
+    "color-no-invalid-hex": true,
+
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always-single-line",
+    "function-comma-space-before": "never",
+    "function-name-case": "lower",
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
+
+    "number-leading-zero": "always",
+    "number-no-trailing-zeros": true,
+    "number-zero-length-no-unit": true,
+
+    "string-no-newline": true,
+    "string-quotes": "double",
```

Comment:
We could change tslint to double-quotes, but I'd think it's a mistake. My vote goes to single quote in CSS and TS. I don't care much about HTML, since inline templates are simpler with double quotes HTML attributes.